<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTemplate</class>
 <widget class="QMainWindow" name="FormTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="form_image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>791</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_field_object">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Field</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_line_edit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>251</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad_Template"/>
    <addaction name="actionSave_Template"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdot">
    <property name="title">
     <string>Run</string>
    </property>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuEdot"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Template">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave_Template">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
